syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -356,7 +372,7 @@ def handle_func_command(cls, command):
             # Check that the function has not been overwritten
```

Comment:
PICKY Stuff: Here the comment should be removed, right? Since we look for ```native_{cmd}```

Reply:
haha no it's good to be picky. here the module `native_torch` defined in native.py overwrites native torch functions... Yeah it's a bit strange, I really had to modify a torch function (aka torch.roll) to behave slightly differently